content/intro-to-storybook/svelte/en/simple-component.md

```diff
@@ -148,16 +152,27 @@ When creating a story we use a base task (`task`) to build out the shape of the
 
 ## Config
 
-We also have to make one small change to the Storybook configuration setup (`.storybook/config.js`) so it notices our `.stories.js` files and uses our CSS file. By default Storybook looks for stories in a `/stories` directory; this tutorial uses a naming scheme that is similar to the `.test.js` or `.spec.js` naming scheme favoured by CRA or Vue CLI for automated tests.
+We'll need to make a couple of changes to the Storybook configuration so it notices not only our recently created stories, but also allow us to use the CSS that was added in the [previous section](/svelte/en/get-started).
+
+Start by changing your Storybook configuration file (`.storybook/main.js`) to the following:
 
 ```javascript
-// .storybook/config.js
-import { configure } from '@storybook/svelte';
-import '../src/index.css';
-configure(require.context('../src/components', true, /\.stories\.js$/), module);
+// .storybook/main.js
+module.exports = {
+  stories: ['../src/components/**/*.stories.js'],
```

Comment:
Should we do this, or just remove the original `src/stories` files?

Reply:
Remove the original stories from the Storybook package? Is that what you mean? If that's the case i think it's probably not a good thing, not only for the user/reader, but also in terms of documentation as it means to make some changes to screenshots and so on. I think leaving a bare bones example in the package is a nice touch to the end user and letting them know with  this small change is a safer route. But i'll leave it to you and the rest of the team on how to proceed

Reply:
Oh, sorry, what I meant was referencing the comment you made on my PR: https://github.com/chromaui/learnstorybook.com/pull/249#discussion_r366488733

If we set `stories: ['../src/**/*.stories.js']`, it will pick up the stories put down by the `sb init` command, right? So we could do that, but just delete those files at this point too.

Reply:
Oh, sorry, my bad, I see that SB sets it to `'../stories/**/*.stories.js', so we'd need to change it in either case.

So the question is: should we change it to `../src/**/*.stories.js' or `../src/components/**/*.stories.js'?.

Reply:
I think we should go with `../src/components/**/*.stories.js` as the majority of the documentation leads the user/reader into that pattern, all of the components used are placed inside the `components` folder. I don't know if it's significant to Storybook itself, but having it just instrospect that folder will tend to speed up the boot process. Don't you think?

Reply:
Sure, ok, let's move forward with that, I'm sure users can figure it out if they want to put stories elsewhere in their own projects.